src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * View: Month View - Single Event Tooltip Date
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ */
+
+$time_format      = tribe_get_time_format();
+$display_end_date = $event->dates->start->format( 'H:i' ) !== $event->dates->end->format( 'H:i' );
```

Comment:
same note as https://github.com/moderntribe/the-events-calendar/pull/2833#discussion_r345563585